providers/enchant_nuspell.cpp

```diff
@@ -0,0 +1,445 @@
+/* enchant
+ * Copyright (C) 2020 Sander van Geloven
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.	 See the GNU
```

Comment:
Something appears to have gone wrong with the spacing here: there should only be two spaces after the full stop!

Reply:
Was unintended change when fixing some spacing in the code. Fixed in https://github.com/nuspell/enchant/commit/205c4b8423d3140a1c6f3e3e77f0841145c7fb69#diff-b4a4b7f548e6522299f21f4f5632ff1cR11